tests/e2e/utils/response_evaluation.py

```diff
@@ -143,3 +151,68 @@ def validate_response(self):
             print("No result. Nothing to process.")
 
         return True
+
+    def evaluate_models(self):
+        """Evaluate models against groundtruth answer."""
+        print("Running model evaluation using groundtruth...")
+        inscope_models = [
```

Comment:
if you use list of lists, no splitting will be needed later. that `+` might be problematic in future when some provider/model choose to use that symbol somewhere (remember c++ :)